os/include/tinyara/pm/pm.h

```diff
@@ -247,11 +247,7 @@
 /* Defines default sleep duration */
 #define DEFAULT_PM_SLEEP_DURATION    10000000 /* 10 sec in microsecond*/
 
-#define PM_LOCK_PATH					"/proc/power/domains/0/pm_lock"
-#define PM_UNLOCK_PATH					"/proc/power/domains/0/pm_unlock"
-#ifdef CONFIG_PM_DVFS
-#define PM_TUNEFREQ_PATH				"/proc/power/domains/0/pm_tunefreq"
-#endif
+#define PM_DRVPATH     "/dev/pm"
```

Comment:
Except the pm.c, does some other file use this? If not, let's move this into the pm.c.